utils/severityutils/severity.go

```diff
@@ -203,21 +203,16 @@ func ParseForDetails(severity string, sarifSeverity bool, applicabilityStatus ja
 	return
 }
 
-func ParseToSeverityDetails(severity string, sarifSeverity, pretty bool, applicabilityStatus jasutils.ApplicabilityStatus) (out formats.SeverityDetails, err error) {
-	parsed, err := ParseSeverity(severity, sarifSeverity)
-	if err != nil {
-		return
-	}
-	return GetSeverityDetails(parsed, applicabilityStatus).ToDetails(parsed, pretty), nil
-}
-
 // -- Getters functions (With default values) --
 
 func GetAsDetails(severity Severity, applicabilityStatus jasutils.ApplicabilityStatus, pretty bool) formats.SeverityDetails {
 	return GetSeverityDetails(severity, applicabilityStatus).ToDetails(severity, pretty)
 }
 
 func GetSeverityDetails(severity Severity, applicabilityStatus jasutils.ApplicabilityStatus) *SeverityDetails {
+	if applicabilityStatus == jasutils.NotScanned {
```

Comment:
what? why assume it's applicable?

Reply:
for appearance but "NotCovered" also applies for this case if we want to change this logic